src/editor.c

```diff
@@ -288,6 +288,21 @@ void editor_snippets_init(void)
 }
 
 
+gboolean motion_notify_event(GtkWidget* widget, GdkEventMotion event, gpointer data)
+{
+	GeanyEditor *editor = data;
+
+	if (event.state & GDK_BUTTON1_MASK && event.state & GDK_MOD1_MASK)
```

Comment:
This includes *any* set of modifiers that contain <kbd>MOD1</kbd>, is that wanted?

```suggestion
	if (keybindings_get_modifiers(event.state) == (GDK_BUTTON1_MASK | GDK_MOD1_MASK))
```

Reply:
Yeah, you're right.

Reply:
`keybindings_get_modifiers(event.state)` filters-out `GDK_BUTTON1_MASK` so it cannot be used for it. What is the correct way to handle this situation? 
```C
if (event.state == (GDK_BUTTON1_MASK | GDK_MOD1_MASK))
```
works for me though it might break if there's some extra stuff in `state`, one could also use
```C
if ((event.state & GDK_BUTTON1_MASK) && keybindings_get_modifiers(event.state) == GDK_MOD1_MASK))
```
but the condition might be satisfied also when there is more than `GDK_BUTTON1_MASK` pressed.